### DISCUSSION TITLE
Containerized Postgrest not connecting to Postgres DB.

### DISCUSSION THREAD

**Discussion by System-Integrator** [PostgREST/postgrest#2248] [Q&A] [answered]
Hi Team,

I am following the section "Containerized PostgREST and db with docker-compose"  as per the link https://postgrest.org/en/stable/install.html#pg-in-docker

But getting below error form the postgrest container.

Attempting to connect to the database...
17/Apr/2022:19:57:01 +0000: {"details":"could not translate host name \"db\" to address: Try again\n","code":"","message":"Database connection error. Retrying the connection."}

Please assist.

Best Regards,

Sujan

**Comment by zzir** [comment]
You can post your `docker-compose.yml` file and see if it changes anything.

**Comment by wolfgangwalther** [comment, marked as answer]
You most likely changed the name of the `db` service in `docker-compose.yml` without adjusting the hostname in the `PGRST_DB_URI` variable.
